### PR TITLE
Changes the display case code

### PR DESCRIPTION
**Pull request by LadyFowls**
https://github.com/Aurorastation/Aurora.3/pull/7673 Revisits this PR

### CODE REVIEW

**Comment by mikomyazaki** [comment]
I have a few issues with this PR:

Codewise:
- Changelog, title and PR text _do not explain the change at all_. The change is that you are locking the spare ID behind another ID requirement inside a box. Please make your PRs more descriptive in future, following the [guidelines here](https://github.com/Aurorastation/Aurora.3/blob/master/.github/CONTRIBUTING.md#changelogs).
- I think you're just reinventing closets (an object that contains other objects that can open and close, and have ID locks). This display case should be a closet subtype instead of its own thing. You can make it only accept ID-type items, or instead only items of tiny size or something.

Balance-wise: 
- This greatly affects antagonist gameplay in relation to the spare ID, and defeats the half of the purpose of the spare ID.

File: maps/aurora/aurora-4_mainlevel.dmm [hunk 1]
```diff
@@ -63932,9 +63932,17 @@
...
 "mKJ" = (
 /turf/simulated/floor/tiled,
 /area/crew_quarters/captain)
+=======
+"mKD" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/structure/trash_pile,
+/turf/simulated/floor/plating,
+/area/maintenance/research_port)
+>>>>>>> master
```
Comment (mikomyazaki) [inline maps/aurora/aurora-4_mainlevel.dmm]: You've accidentally committed some merge conflict resolution stuff. Need to deal with that. Not sure why that can even pass travis' tests.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
This isn't something that will pass like this. The spare being stolen isn't something that should be told to everyone at all. It'll just lead to antags getting assblasted far sooner.

**Comment by skull132** [comment]
Dead in the water.

**Pull request closed without merging**